Resources/public/js/views/services/ez-testoneviewservice.js

```diff
@@ -0,0 +1,220 @@
+                        currentLocation: {
+                            locationId: data.article.locationId,
+                            name: data.article.name,
+                            hasChildren: true
+                        }
+                    });
+                });
+            } else {
+                articlesView.fire('articleSelected', {
+                    article: data.article
+                });
+            }
+        },
+
+        /**
+         * Event handler for loading intro
+         *
+         * @method _loadIntro
+         * @param {Object} e event facade
+         * @protected
+         */
+        _loadIntro: function (e) {
+            var introView = e.target,
+                contentService = this.get('capi').getContentService(),
+                locationId = e.locationId,
+                intro;
+
+            /**
+             * Callback for processing loaded content
+             *
+             * @param {boolean} err
+             * @param {Object} response
+             * @protected
+             */
+            function loadContent(err, response) {
+                if (err) {
+                    introView.fire('introLoadFailed');
+                } else {
+                    response.document.Version.Fields.field.forEach(function (fieldDefinition) {
+                        if (fieldDefinition.fieldDefinitionIdentifier === 'intro') {
+                            intro = new Y.eZ.XmlTextView({
+                                field: fieldDefinition,
+                                fieldDefinition: {
+                                    fieldType: 'xmltext'
+                                }
+                            });
+                        }
+                    });
+
+                    introView.fire('introLoaded', {
+                        intro: intro
+                    });
+                }
+            }
+
+            introView.fire('introLoading');
+            contentService.loadLocation(locationId, function (err, response) {
+                if (err) {
+                    introView.fire('introLoadFailed');
+                }
+                else {
+                    contentService.loadCurrentVersion(response.document.Location.Content._href, loadContent);
+                }
+            });
+        },
+
+        /**
+         * Sets the default root location for article list
+         *
+         * @method _getViewParameters
+         * @returns {{service: Y.eZ.TestOneViewService}}
+         * @proteced
+         */
+        _getViewParameters: function () {
+            return {
+                rootLocation: this.get('capi').getContentService()._rootPath + 'content/locations/1/2'
+            };
+        },
+
+        /**
+         * Loads article list from capi
+         *
+         * @method loadArticleList
+         * @param {String} locationId
+         * @param {Function} callback
+         */
+        loadArticleList: function (locationId, callback) {
```

Comment:
To chyba powinno być jako protected. Generalnie, serwisy nie powinny udostępniać publicznych metod (tak wiem, że w JS nie ma takiego podziału na metody publiczne, chronione i prywatne).